pkgs/applications/networking/giara/default.nix

```diff
@@ -6,17 +6,19 @@
 , ninja
 , python3
 , wrapGAppsHook
-, gtk3
+, gtk4
 , gdk-pixbuf
 , webkitgtk
-, gtksourceview4
+, gtksourceview5
 , libhandy
```

Comment:
Maybe libhandy is no longer needed?

Reply:
True, it builds without handy as well. Will remove it. Thanks